lib/aws_codegen/docstring.ex

```diff
@@ -36,6 +36,9 @@ defmodule AWS.CodeGen.Docstring do
     |> fix_long_break_lines()
     |> String.trim_trailing()
     |> String.replace(@two_break_lines, "\n%%\n")
+    |> String.replace(~r/&#39;/, "'")
+    |> String.replace("`AVAILABLE`", "`AVAILABLE'") # aws-sdk-go docs are broken for this, hack it to make the edocs work
```

Comment:
the \`AVAILABLE' thing is peculiar to edoc from way back before markdown was ever a thing